examples/react/src/contexts/WalletSelectorContext.tsx

```diff
@@ -85,7 +59,17 @@ export const WalletSelectorContextProvider: React.FC = ({ children }) => {
     });
     const _modal = setupModal(_selector, { contractId: CONTRACT_ID });
     const state = _selector.store.getState();
-    syncAccountState(localStorage.getItem("accountId"), state.accounts);
+
+    if (!state.accounts.length) {
+      setAccounts([]);
+    }
+
+    const newAccount = state.accounts.find((account) => account.active);
+
+    if (newAccount) {
+      _selector.setActiveAccount(newAccount.accountId);
+      setAccounts(state.accounts);
+    }
```

Comment:
We should be able to replace this (and in the subscription further down) with simply:

```suggestion
setAccounts(state.accounts);
```